libbeat/cfgfile/reload.go

```diff
@@ -215,10 +218,11 @@ func (rl *Reloader) Run(runnerFactory RunnerFactory) {
 
 			debugf("Number of module configs found: %v", len(configs))
 
-			if err := list.Reload(configs); err != nil {
-				// Make sure the next run also updates because some runners were not properly loaded
-				overwriteUpdate = true
+			if err = list.Reload(configs); err != nil {
+				logp.Err("Error reloading module configurations: %v", err)
```

Comment:
Please check if we should really log this on the error level. Could this reload return an error because a harvester was not finished yet? I just want to make sure we don't spam the error logs with this in some cases.

Reply:
Looking at the test runs, the results are real errors, but they're also already logged by `Reload` itself, so this does end up being noisy and redundant. Removed / commented.